themes/mlibrary_new/custom.php

```diff
@@ -349,37 +358,16 @@ function mlibrary_metadata_sideinfo($item){
     );
 
     if (!empty($elementTexts)) {
-        $name = ($elementName == 'Identifier') ? 'Item Source' : $elementName;
+        $name = ($elementName == 'Identifier' ? 'Source' : ($elementName == 'Source' ? 'Item Source' : $elementName));
```

Comment:
Rather than doing this every time, I recommend putting it the display label in `$elementInfos` array.